tuf/keys.py

```diff
@@ -223,6 +233,9 @@ def generate_rsa_key(bits=_DEFAULT_RSA_KEY_BITS):
   if _RSA_CRYPTO_LIBRARY == 'pycrypto':
     public, private = tuf.pycrypto_keys.generate_rsa_public_and_private(bits)
   
+  elif _RSA_CRYPTO_LIBRARY == 'pyca-cryptography':
```

Comment:
Not resolved in PR #311. (Seems worth doing to my naive eyes. Any issues with it?)